finix/model/finix_list.py

```diff
@@ -0,0 +1,171 @@
+from typing import List
+
+
+# finix list implementation
+class FinixList:
+    def __init__(self, some_list, method, **kwargs):
```

Comment:
`some_list` is a bit confusing to me. I see there is `some_list.embedded` and `some_list.links` below, but I'm not 100% sure on what object `some_list` is.

Reply:
some_list is the old response of list requests, which has many possible types such as TransferList, AuthorizationList, etc.